client/v2/common/models/shim.go

```diff
@@ -17,3 +19,6 @@ type Supply SupplyResponse
 
 // VersionBuild
 type VersionBuild BuildVersion
+
+// LedgerStateDelta aliases to the manually defined type.
+type LedgerStateDelta types.LedgerStateDelta
```

Comment:
I hadn't used this trick before, but it's the reason no custom generation was needed for `getLedgerStateDelta.go`. I'm not sure why the generator didn't create a `model.LedgerStateDelta` object.